pct.sh

```diff
@@ -70,10 +146,4 @@ elif grep -q -F -e '<status>TEST_FAILURES</status>' pct-report.xml; then
 	done
 fi
 
-# TODO various problems with PCT itself (e.g. https://github.com/jenkinsci/bom/pull/338#issuecomment-715256727)
-# and anyway the tests in PluginAutomaticTestBuilder are generally uninteresting in a PCT context
-# We always try to run this test rather than adding it to excludes.txt in order
-# to be able to detect if PCT failed to run tests at all a few lines above.
-rm -fv pct-work/*/{,*/}target/surefire-reports/TEST-InjectedTest.xml
```

Comment:
No longer needed as of https://github.com/jenkinsci/maven-hpi-plugin/pull/336.

Reply:
Wow, OK. We should be comfortable readding this exclusion at the first sign of trouble, though—this test suite really does not add value in the PCT context.